blocks/hero/hero.js

```diff
@@ -42,7 +43,7 @@ export default async function decorate(block) {
   const picture = block.querySelector('picture');
   block.innerHTML = template(
     {
-      Date: dateString,
+      Date: dateString.toLocaleDateString('de-DE', options),
```

Comment:
Thanks @RitwikSrivastava for chipping in here. Instead of explicitly adding the locale here, you can perhaps fetch it using `getMetadata('locale')` method. The `getMetadata` function is defined in the `franklin-lib`